lib/test.rb

```diff
@@ -748,6 +780,11 @@ def formula(formula_name)
 
         @source_dependents.each do |dependent|
           install_dependent_from_source(dependent)
+
+          bottled = with_env(HOMEBREW_SKIP_OR_LATER_BOTTLES: "1") do
+            dependent.bottled?
+          end
+          install_bottled_dependent(dependent) if bottled
```

Comment:
This is a separate change, and a separate commit. But one it makes sense to do at the same time to ensure test coverage remains high.

When we build from source, we don't actually properly test linkage of existing bottles. For example, if OCaml is updated, the tests will currently never detect that ocaml-findlib needs a revision bump because the existing bottles are never tested. This change make it run _both_ a build-from-source and a bottle install test.